iina/ThumbnailCache.swift

```diff
@@ -186,6 +191,15 @@ class ThumbnailCache {
     return result
   }
 
+  static private func delete(_ pathURL: URL) {
```

Comment:
```suggestion
  private static func deleteCacheFile(at pathURL: URL) {
```

Reply:
Fixed.
I changed the ordering of keywords throughout the class to be "private static", which I prefer as well. I had reversed it to be consistent with the existing code.

I actually searched around for official guidelines on preferred ordering of keywords and didn't find any for Swift. Java allows you to use the order you want, but does have an official preferred order. The Eclipse IDE source formatter could be configured to automatically put your keywords in the official order. I miss that.

Reply:
Yeah, I don't blame you, I just think the other ordering was worse I figured we might as well switch it around for new code :)

Reply:
Always feel free to ask me to cleanup style issues like this. I'm happy to do it.